sources/is.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "url": "https://api.ust.is/aq/a/getLatest",
+        "adapter": "ust-ist",
+        "name": "Umhverfisstofnun",
+        "country": "IS",
+        "description": "Environment Agency of Iceland",
+        "sourceURL": "https://ust.is",
+        "contacts": [
```

Comment:
@Stalfur do you mind being the contact or know someone who should be? If not, we can put down `info@openaq.org`

Reply:
We have a loftgaedi@umhverfisstofnun.is (shared mailbox) that should be used.

Reply:
Great, was added. Thanks